airbyte-ci/connectors_ci/metadata_service/lib/metadata_service/models/src/AllowedHosts.yaml

```diff
@@ -10,4 +10,4 @@ properties:
     type: array
     description: An array of hosts that this connector can connect to.  AllowedHosts not being present for the source or destination means that access to all hosts is allowed.  An empty list here means that no network access is granted.
     items:
-      type: string
```

Comment:
why this change?

Reply:
This was an automated change. I'm not sure why it came up.